content/show/coder-radio/_index.md

```diff
@@ -12,8 +12,13 @@ active = true
 
 header_image = "/images/shows/coder-radio.png"
 
+
+support_link = "https://jupitersignal.memberful.com/checkout?plan=53334"
+subscribe_link = "https://new.jupiterbroadcasting.com/show/coder-radio/subscribe/"
+
 support_link = "/membership"
```

Comment:
```suggestion
```

Reply:
^ deletes one of the `support_link` references, changes it to `/membership` but only for Coder. Likely need to apply this to all shows...